### DISCUSSION TITLE
[Home] signs

### DISCUSSION THREAD

**Discussion by melaniebeedot** [EssentialsX/Essentials#5610] [Ideas and Feature Suggestions] [answered]
Would it be feasible to have [Home] signs that a player could make that connect to their /homes, similar to how [Warp] signs connect to /warps? We've created something makeshift on our server using ServerSigns, which essentially is just a sign that looks like a [Warp] sign but it says [Home] and a home name below and when the players click it, it runs the command /home homename and sends them to their respective homes (if they have a home set by that name). But it would be nice if this was part of Essentials instead so the players could create these signs for themselves, for personal convenience only, really, but still. (We can't allow them to use ServerSigns themselves because they could very easily abuse it and put any command they wanted on the sign, the perms for that plugin are kinda all or nothing..) but I digress. Thanks for reading my idea!

**Comment by JRoy** [comment, marked as answer]
Homes are private to a given user, a sign which only you could click doesn't fit the use case for Essentials' sign system.
